__tests__/api/delete_loan.js

```diff
@@ -0,0 +1,9 @@
+const frisby = require('../../test_config');
```

Comment:
`frisby` should change, it should be another name, because you have modified frisby its self...